enterprise/actions.py

```diff
@@ -7,6 +7,7 @@
 
 import unicodecsv
 from six import string_types
+
```

Comment:
@asadiqbal08 nit: might be good to remove this newline - the file is not otherwise affected